### ISSUE TITLE
Missing License Information

### ISSUE THREAD

**Issue by sagikazarmark** [alexsocha/luckyblock#11]
When trying to install this mod using the latest MC and MC Forge version, I get the following error:

```
Missing License Information in file Mod File: /data/mods/luckyblock-1.16.1-1.jar
```

Looks like as of 1.6.2 license is a required field.

References:

- https://github.com/MinecraftForge/MinecraftForge/issues/7184
- https://github.com/itzg/docker-minecraft-server/issues/643

**Comment by alexsocha** [comment]
Thanks! Fixed in the latest 1.16.4-1 version, which should be backwards-compatible

**Comment by sagikazarmark** [comment]
Thanks!
